cmd/av/pr_status.go

```diff
@@ -0,0 +1,218 @@
+			"\n",
+		)
+		_, _ = fmt.Fprint(os.Stderr, indent, "Status: ", colors.UserInput(prStatus), "\n")
+
+		if prStatus == "PENDING" || prStatus == "BLOCKED" {
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				"Status reason: ",
+				colors.UserInput(query.GithubRepository.PullRequest.StatusReason),
+				"\n",
+			)
+		}
+
+		_, _ = fmt.Fprint(
+			os.Stderr,
+			indent,
+			"Author: ",
+			colors.UserInput(query.GithubRepository.PullRequest.Author.Login),
+			"\n",
+		)
+		_, _ = fmt.Fprint(
+			os.Stderr,
+			indent,
+			"Created at: ",
+			colors.UserInput(formatTimestamp(string(query.GithubRepository.PullRequest.CreatedAt))),
+			"\n",
+		)
+
+		if prStatus == "QUEUED" {
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				"Queued at: ",
+				colors.UserInput(
+					formatTimestamp(string(query.GithubRepository.PullRequest.QueuedAt)),
+				),
+				"\n",
+			)
+		}
+		if prStatus == "MERGED" {
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				"Merged at: ",
+				colors.UserInput(
+					formatTimestamp(string(query.GithubRepository.PullRequest.MergedAt)),
+				),
+				"\n",
+			)
+		}
+
+		_, _ = fmt.Fprint(
+			os.Stderr,
+			indent,
+			"Base branch: ",
+			colors.UserInput(
+				query.GithubRepository.PullRequest.BaseBranchName,
+				" <- ",
+				query.GithubRepository.PullRequest.HeadBranchName,
+			),
+			"\n\n",
+		)
+
+		// Required checks section
+		_, _ = fmt.Fprint(os.Stderr, "Required Checks\n")
+		requiredCheckStatuses := query.GithubRepository.PullRequest.RequiredCheckStatuses
+		for index := range requiredCheckStatuses {
+			result := requiredCheckStatuses[index].Result
+			requiredCheckName := requiredCheckStatuses[index].RequiredCheck.Pattern
+			_, _ = fmt.Fprint(
+				os.Stderr,
+				indent,
+				emojiForRequiredCheckResult(string(result)),
+				" ",
+				colors.UserInput(requiredCheckName),
+				"\n",
+			)
+		}
+
+		return nil
+	},
+}
+
+func getQueryVariables() (map[string]interface{}, error) {
+	repo, err := getRepo()
+	if err != nil {
+		return nil, err
+	}
+
+	db, err := getDB(repo)
+	if err != nil {
+		return nil, err
+	}
+
+	tx := db.ReadTx()
+
+	currentBranchName, err := repo.CurrentBranchName()
+	if err != nil {
+		return nil, err
+	}
+
+	branch, exists := tx.Branch(currentBranchName)
+	if !exists {
+		return nil, errors.New("could not find current branch")
+	}
+
+	if branch.PullRequest == nil {
+		return nil, errors.New("pull request does not exist")
+	}
+
+	prNumber := branch.PullRequest.Number
+	repository, _ := tx.Repository()
+	var variables = map[string]interface{}{
+		"repoOwner": graphql.String(repository.Owner),
+		"repoName":  graphql.String(repository.Name),
+		"prNumber":  graphql.Int(prNumber),
+	}
+	return variables, nil
+}
+
+func emojiForRequiredCheckResult(result string) string {
+	if result == "SUCCESS" {
+		return "\u2705"
+	} else if result == "FAILURE" {
+		return "\u274C"
+	} else {
+		return "\u231B"
+	}
+}
+
+func formatTimestamp(timeString string) string {
```

Comment:
We should probably have a package that just contains utilities like this. Maybe like `utils/timeutils` so that we have `timeutils.FormatLocal(t time.Time) string`.